modules/nw-rfhe-creating-hardware-event-kind-cr.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
+
+[id="nw-rfhe-hardware-event-kind-cr_{context}"]
+= Creating the HardwareEvent Kind CR
```

Comment:
I prefer the title:
Creating the `HardwareEvent` resource

Reply:
Done.